android/src/main/java/so/onekey/lib/ble/utils/BleUtilsModule.kt

```
@@ -32,7 +32,7 @@ class BleUtilsModule(private val reactContext: ReactApplicationContext) :
   private fun getBluetoothManager(): BluetoothManager? {
     if (bluetoothManager == null) {
       bluetoothManager =
-        reactContext.getSystemService(Context.BLUETOOTH_SERVICE)
+        reactContext.getSystemService(Context.BLUETOOTH_SERVICE) as BluetoothManager?
```

detekt (1.23.7)

[warning] Use the safe cast ('as? BluetoothManager') instead of 'as BluetoothManager?'.

(detekt.potential-bugs.CastToNullableType)